addons/mail/static/src/components/discuss/discuss.xml

```diff
@@ -43,8 +43,7 @@
         <t t-if="discussView.discuss.threadView">
             <t name="beforeThread"/>
             <ThreadView
-                className="'o_Discuss_thread'"
-                t-att-class="{ 'o-mobile': messaging.device.isMobile }"
+                className="'o_Discuss_thread' + (messaging.device.isMobile ? ' o-mobile' : '')"
```

Comment:
We have `classNameObj` prop to resolve this issue without having to do string manipulations.